docs/guides/1.19.0-upgrade-guide.md

```diff
@@ -0,0 +1,31 @@
+---
+page_title: "Upgrade Guide 1.19.0"
+---
+
+# MongoDB Atlas Provider 1.19.0: Upgrade and Information Guide
+
+The Terraform MongoDB Atlas Provider version 1.19.0 has a number of new and exciting features.
+
+**New Resources, Data Sources, and Features:**
+- You can now [manage customer keys from Azure Key Vault over Private Endpoints](https://www.mongodb.com/docs/atlas/security/azure-kms-over-private-endpoint/#manage-customer-keys-with-azure-key-vault-over-private-endpoints) to further encrypt your data at rest in Atlas with the new `mongodbatlas_encryption_at_rest_private_endpoint` resource and data sources in conjunction with the existing `mongodbatlas_encryption_at_rest` resource.
+    - In order to configure a private endpoint for your Azure Key Vault, the corresponding `mongodbatlas_encryption_at_rest` resource has to be adjusted by configuring `azure_key_vault_config.require_private_networking` to `true`. This attribute can be updated in place, ensuring the customer-managed keys encryption is never disabled.
+    - To learn more, please review `mongodbatlas_encryption_at_rest_private_endpoint` [resource documentation](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/encryption_at_rest_private_endpoint).
+
+- You can now use the new `mongodbatlas_project_ip_addresses` data source that returns the IP addresses in an Atlas project categorized by services.
+
+- You can now manage [Atlas Stream Processors](https://www.mongodb.com/docs/atlas/atlas-stream-processing/overview/) with the new `mongodbatlas_stream_processor` resource, `mongodbatlas_stream_processor` and `mongodbatlas_stream_processors` data sources. To learn more, please review `mongodbatlas_stream_processor` [resource documentation](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/stream_processor).
```

Comment:
second link does not yet exists, but will once the documentation for 1.19.0 is published in the registry